docs/index.asciidoc

```diff
@@ -101,6 +101,10 @@ Notice also that when you use `query_template`, the Logstash attributes `result_
 and `sort` will be ignored. They should be specified directly in the JSON
 template, as shown in the example above.
 
+[id="plugins-{type}s-{plugin}-auth"]
+==== Authentication
+
+Authentication to a secure Elasticsearch cluster is possible using one of the `user`/`password`, `cloud_auth` or `api_key` options.
```

Comment:
```suggestion
Authentication to a secure Elasticsearch cluster is possible using _one_ of the following options:
* <<plugins-{type}s-{plugin}-user>> AND <<plugins-{type}s-{plugin}-password>>
* <<plugins-{type}s-{plugin}-cloud_auth>>
* <<plugins-{type}s-{plugin}-api_key>>
```

Reply:
I think this section can be improved by linking to the specific options.

Reply:
@karenzone  WDYT?

Reply:
I think it makes sense.  Anything we can do to help users get the info they need and back to work faster is a good thing.

Reply:
But the above formatting is incorrect.  The new content won't render correctly unless you add a return after the colon on line 107.

Reply:
Added the newline to the suggestion.